### ISSUE TITLE
Honeywell JSON parse error

### ISSUE THREAD

**Issue by wongey** [ORNL-QCI/xacc#32]
Upon sending a program to honeywell:H1-1E, a new error is reported:

```
Failed to execute XACC: [json.exception.parse_error.101] parse error at line 1, column 1: syntax error while parsing value - invalid literal; last read: '<U+001F>' 
```

**Comment by wongey** [comment]
I believe this issue is related to to spack library nlohmann@3.11.3 (which was recently updated from 3.1 on my end) after the recent merge [PR#30](https://github.com/ORNL-QCI/xacc/pull/30).

cc: @sethrj

**Comment by wongey** [comment]
Update: This becomes a non-issue when building outside of the spack environment.

But will keep this issue open for now as @sethrj investigates this for the spack environment.

**Comment by sethrj** [comment]
That makes no sense... can you try spack again? maybe it was just a fluke? It's impossible to tell because (until my PR) there's no way to get the failed server response other than running through a debugger 😠

**Comment by wongey** [comment]
Yes: See [my comment here in PR #33](https://github.com/ORNL-QCI/xacc/pull/33#issuecomment-2274223403).
